doc/programming_guide/ffva/deploying/native_windows.rst

```diff
@@ -48,7 +48,7 @@ Run the following commands in the root folder to build the USB firmware:
 
     cmake -G "NMake Makefiles" -B build -DCMAKE_TOOLCHAIN_FILE=xmos_cmake_toolchain/xs3a.cmake
     cd build
-    nmake example_ffva_ua_adec
+    nmake example_ffva_ua_adec_altarch
```

Comment:
If you're updating the documentation, then please also change it from using nmake to ninja, and be sure to test that the instructions actually work with ninja.

Reply:
Good point. I made the changes and I replaced `-DCMAKE_TOOLCHAIN_FILE` with `--toolchain` too.